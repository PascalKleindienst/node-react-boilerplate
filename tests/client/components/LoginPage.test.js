// ==============================================
// LOGIN PAGE TEST ==============================
// ==============================================
import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../../src/components/LoginPage';

describe('my Login page component', () => {
    test('should render correctly', () => {
        const wrapper = shallow(<LoginPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
