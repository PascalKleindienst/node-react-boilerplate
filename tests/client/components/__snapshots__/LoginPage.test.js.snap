// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my Login page component should render correctly 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      Boilderplate
    </h1>
    <p>
      Tagline for App
    </p>
    <p>
      <a
        className="button"
        href="/auth/twitter"
      >
        Login with Twitter
      </a>
    </p>
    <p>
      <a
        className="button"
        href="/auth/facebook"
      >
        Login with Facebook
      </a>
    </p>
    <p>
      <a
        className="button"
        href="/auth/google"
      >
        Login with Google
      </a>
    </p>
  </div>
</div>
`;
